releases.yml

```diff
@@ -3,7 +3,10 @@
     - component: Common grid patterns
       url: /docs/patterns/grid#common-grid-patterns
       status: New
-      notes: "We've added new classes: <code>.row--50-50</code>, <code>.row--25-75</code>, <code>.row--25-25-50</code>, <code>.row--25-25-25-25</code> as an alias for common grid layouts."
```

Comment:
@ClementChaumel Well… this should not get removed. I guess we need another fix to this yaml file ;)